README.md

```diff
@@ -1 +1,79 @@
-# ethereum-explorer-application
```

Comment:
- [ ] Great job documenting your project! It would be great if you showed some of your code in the demo video and highlight a part of the code that you liked.

Reply:
Hi @smunozmo , thank you for pointing this out, I have fixed this issue. I used the if the approach to filter the ascending and descending order. These other comes directly from the API, so I made the API parameter to sort them, as a variable and passed either ascending or descending order to the API, to filter out the data. I have also updated the demo video as instructed.

Thanks.

Reply:
@Fadahunsiseyi great work! Your app is ready for your portfolio 😎, congrats!